pkg/cloudprovider/providers/openstack/openstack_loadbalancer.go

```diff
@@ -75,6 +75,9 @@ const (
 	ServiceAnnotationLoadBalancerSubnetID          = "loadbalancer.openstack.org/subnet-id"
 	ServiceAnnotationLoadBalancerConnLimit         = "loadbalancer.openstack.org/connection-limit"
 
+	// annotation without value, if key defined it will keep floatingip. If not defined it will delete floatingip when lb is deleted
+	ServiceAnnotationLoadBalancerKeepFloatingIP = "loadbalancer.openstack.org/keep-floatingip"
```

Comment:
IMO, if user specify the fip, don't delete this fip, or delete it.
But it is hard to do...

Reply:
it is not possible to define fip currently?

Reply:
I mean that cloud provider just delete the floatingip which created by kubernetes,
if user specify a exist floatingip, cloud provider don't delete it.